src/Navigation.ts

```diff
@@ -0,0 +1,153 @@
+﻿class Navigation {
+	private table: Table;
+
+	private rowNum: number;			// The first Id to be fetched.
+	private numToFetch: number;		// The last Id to be fetched.
+	private searchedId: number;
+	private maxRecords: number;
+
+	private headerRowHeight = 23;
```

Comment:
no types here, but variables above have them declared. Should keeps things consistent.